google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -172,9 +197,9 @@ public Builder(HttpResponse response) {
       this(response.getStatusCode(), response.getStatusMessage(), response.getHeaders());
       // content
       try {
-        content = response.parseAsString();
-        if (content.length() == 0) {
-          content = null;
+        InputStream inputStream = response.getContent();
+        if (inputStream != null) {
+          content = ByteStreams.toByteArray(response.getContent());
```

Comment:
nit: this should just be `ByteStreams.toByteArray(inputStream)`

Reply:
thanks